processors/base/helpers/__init__.py

```diff
@@ -4,20 +4,28 @@
 from __future__ import print_function
 from __future__ import unicode_literals
 
+import os
 import re
 import json
 import uuid
 import string
 import logging
 import datetime
 import urlparse
+import csv
+import iso3166
+
+from fuzzywuzzy import process, fuzz
```

Comment:
Keep this import in the above section, just after the `iso3166`, and use `import fuzzywuzzy` instead of `from foo import bar`